README.md

```diff
@@ -57,6 +57,21 @@ A diagram of this running 5 steps on a single sample image may help explain how
 ## Installation
 
 This library conveniently works in the browser, in Node, and on the command line (CLI).
+To install the app, run the following commands in a command line:
+
+* `git clone https://github.com/vaibhavmatta/image-sequencer`
```

Comment:
also the url link is specific to your fork, which in not correct for new contributors.

Reply:
My Bad 😅. I will add the original repository link.